gfx/2d/ScaledFontMac.cpp

```diff
@@ -524,17 +524,30 @@ static CFDictionaryRef CreateVariationDictionaryOrNull(
   // This will typically be a very low value, so we just linear-search them.
   bool allDefaultValues = true;
 
+  // https://github.com/RecordReplay/backend/issues/4555
+  recordreplay::RecordReplayAssert("CreateVariationDictionaryOrNull #5 %d", (int)axisCount);
```

Comment:
This and the following changes seem to have leaked from another patch?  Or are you just lumping them in here for convenience?

Reply:
This is for convenience, see https://github.com/RecordReplay/backend/issues/4555